tutorials/3d/procedural_geometry/surfacetool.rst

```diff
@@ -60,14 +68,26 @@ Code creates a triangle with indices
     var mesh = st.commit()
 
 You can optionally add an index array, either by calling ``add_index()`` and adding
-vertices to the index array or by calling ``index()`` which shrinks the vertex array
-to remove duplicate vertices.
+vertices to the index array manually, or by calling ``index()`` once,
+which generates the index array automatically and
+shrinks the vertex array to remove duplicate vertices.
```

Comment:
I think the definition of duplicate needs to be called out here. With regards to:

- which vertex properties? Colors? Normals? 
- how is floating point rounding dealt with?

The only way I would have found this info is by reading the linked issue/source.

Reply:
hi,

unfortunately I don't have such deep insights into godot so I'm not sure how e.g. rounding is handled. it would be good if someone else could take this over